cypress/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "baseUrl": "../",
+    "paths": {
+      "~/*": ["src/*"]
+    },
+    "target": "es5",
+    "lib": ["es5", "dom"],
```

```suggestion
    "lib": ["dom", "esnext"],
    "module": "esnext",
```

Not sure why you'd want to limit the cypress source to es5. In addition I think you might need the module setting for named imports to work properly.